algolia.js

```diff
@@ -55,15 +57,21 @@ const chunkAlgolia = (arr) => {
   let tempSizeInBytes = 0;
   let temp = [];
   for (const arrItem of arr) {
-    const current = trimBytes(arrItem);
-    const currentSizeInBytes = byteof(current);
-
-    if (tempSizeInBytes + currentSizeInBytes < maxChunkSizeInBytes) {
-      temp.push(current);
+    const currentSizeInBytes = byteof(arrItem);
+
+    if (currentSizeInBytes >= maxChunkSizeInBytes) {
+      throw new Error(
+        `${path.join(
+          arrItem.locale || '',
+          arrItem.url || '',
+        )} is >= than 10 MB`,
```

Comment:
Nit: it might be worth creating a `maxChunkSizeInMiB` (or something like that) and using that to generate the error message so it stays in sync with the logic.